application/app/src/test/java/uk/nhs/digital/uec/api/integration/fuzzysearch/FuzzySearchSuccessTest.java

```diff
@@ -16,13 +16,15 @@
 import org.springframework.http.HttpMethod;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.ActiveProfiles;
 import org.springframework.web.util.UriComponentsBuilder;
 import uk.nhs.digital.uec.api.model.ApiSuccessResponse;
 import uk.nhs.digital.uec.api.model.DosService;
+import uk.nhs.digital.uec.api.util.MockDosServicesUtil;
 import uk.nhs.digital.uec.api.util.PropertySourceResolver;
-import uk.nhs.digital.uec.api.utils.MockDosServicesUtil;
 
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+@ActiveProfiles("test")
```

Comment:
Shouldn't the annotations go below the class comment? Can we also check all the other files, please?

Reply:
Yes they should I'll action.